ProjectOptions.cmake

```diff
@@ -53,9 +53,9 @@ macro(myproject_setup_options)
     option(myproject_ENABLE_SANITIZER_UNDEFINED "Enable undefined sanitizer" ${SUPPORTS_UBSAN})
     option(myproject_ENABLE_SANITIZER_THREAD "Enable thread sanitizer" OFF)
     option(myproject_ENABLE_SANITIZER_MEMORY "Enable memory sanitizer" OFF)
-    option(myproject_ENABLE_UNITY_BUILD "Enable unity builds" OFF)
-    option(myproject_ENABLE_CLANG_TIDY "Enable clang-tidy" ON)
-    option(myproject_ENABLE_CPPCHECK "Enable cpp-check analysis" ON)
+    option(myproject_ENABLE_UNITY_BUILD "Enable unity builds" ON)
+    option(myproject_ENABLE_CLANG_TIDY "Enable clang-tidy" OFF)
+    option(myproject_ENABLE_CPPCHECK "Enable cpp-check analysis" OFF)
```

Comment:
**IMHO**: developer shouldn't wast its time.

`run-clang-tidy -p out/builddir` is easy to use

Only on **CI** `clang-tidy` must at least run one times.